docs/artifacts/where-the-plugin-lives.html

```
@@ -0,0 +1,156 @@
+    below were wrong on Forks 1 and 2 and are left in place so the record shows what was proposed.</p>
+  </div>
+  <figure>
+<pre class="mermaid">
+%%{init: {'theme':'base','themeVariables':{'primaryColor':'#f2f2f4','primaryTextColor':'#1b1b1b','primaryBorderColor':'#8d8d94','lineColor':'#4a4a4a','fontFamily':'IBM Plex Mono, monospace','fontSize':'12px'}}}%%
+flowchart LR
+  A["claude plugin marketplace add ray-manaloto/claude-code-marketplace"] -->|"reads .claude-plugin/marketplace.json"| M["marketplace.json entry · aggregated-research"]
+  M -->|"source: FORK 1"| P["plugin dir · plugin.json · skills/ · hooks/hooks.json · .lsp.json · bin/"]
+  P -->|"claude plugin install … --yes, + 4 declared deps"| I["installed plugin"]
+  I -->|"SessionStart hook: FORK 2"| C["CLI in $CLAUDE_PLUGIN_DATA · aggregated-research trackers …"]
+  I -->|"skill runs via Bash"| C
+  C -->|"--out .research/#lt;slug#gt;.json"| O[(".research/ files")]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)